docs/monitors/kubelet-stats.md

```diff
@@ -38,7 +38,7 @@ The **nested** `kubeletAPI` config object has the following fields:
 
 | Config option | Required | Type | Description |
 | --- | --- | --- | --- |
-| `url` | no | `string` | URL of the Kubelet instance.  This will default to `https://<current node hostname>:10250` if not provided. |
+| `url` | no | `string` | URL of the Kubelet instance.  This will default to `https://<current node hostname>:10255` if not provided. |
```

Comment:
Typically port 10255 is HTTP, not HTTPS.

Reply:
updated